### DISCUSSION TITLE
COPC Cloud Optimized Point Cloud

### DISCUSSION THREAD

**Discussion by Hutch07** [iTowns/itowns#2086] [Q&A]
I have started Using Hobu's Cloud Optimized Point Cloud on LAZ files.  Is this supported by ITowns? https://github.com/copcio/copcio.github.io

**Comment by mgermerie** [comment]
Hi ! COPC support is brought by #2110. It should be available in `itowns@next` npm package soon and in `itowns` package in `2.41` or `2.42`.
